components/admin_console/license_settings/license_settings.jsx

```diff
@@ -71,7 +71,9 @@ export default class LicenseSettings extends React.PureComponent {
             this.reloadPercentage();
         }
         this.props.actions.getLicenseConfig();
-        this.props.actions.getPrevTrialLicense();
+        if (this.props.enterpriseReady) {
+            this.props.actions.getPrevTrialLicense();
+        }
```

Comment:
If the order of these calls is not important I can just add the call to an else block. Let me know.

Reply:
I don't think the order of these calls matter, you could move it into the else if you want